bundles/org.openhab.binding.irobot/src/main/resources/OH-INF/config/thing.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:irobot:thing">
+		<parameter name="address" type="text">
+			<context>network-address</context>
+			<label>Network Address</label>
+			<description>Network address of the robot</description>
+			<required>true</required>
+		</parameter>
+		<parameter name="family" type="text">
+			<label>IRobot Family</label>
+			<description>IRobot device family</description>
```

Comment:
If I see correctly this is newly introduced. If you need to differentiate between models, you could define a thing type for each model.

Reply:
removed the parameter family, as it is not yet used. We can add it again as soon as it is needed.